src/lib/Serialize.hs

```diff
@@ -79,6 +91,18 @@ prettyVal val = case val of
             variant = Variant (NoExt types) theLabel repeatNum value
         _ -> error "SumAsProd with an unsupported type"
     _ -> return $ pretty con
+  Record (LabeledItems row) -> do
```

Comment:
Is this to fix the record-of-tables issue? Or did you have some other reason to support this?

Reply:
Yes, this is to fix the record-of-tables issue, which was merged in https://github.com/google-research/dex-lang/pull/327. I've rebased.